.eslintrc

```diff
@@ -179,7 +178,6 @@
     "one-var": 0,
     "operator-assignment": 0,
     "padded-blocks": 0,
-    "quote-props": 0,
```

Comment:
no-extra-parens is found twice in that file. `"quote-props": 0` and `    "quote-props": [2, "as-needed", {"keywords": true}]`. The upgrade of grunt-eslint from 17.1.0 -> 19.0.0 identified this.